src/app/utils/tests/utils.spec.ts

```diff
@@ -70,8 +71,10 @@ describe('utils', () => {
         });
     });
     describe('assert', () => {
-        it('Should throw when condition is false', () => {
+        it('Should log error and throw when condition is false', () => {
```

Comment:
perhaps avoiding using "log" as a way to save "save in database", cause it's probably mostly evokationg console.log

Reply:
non ? Log dans le sens logger, noter qu'il s'est passé quelque chose, comme les marins au 18ème siècles qui avaient leur log (littéralement journal de bord). Log pour moi m'évoque pas trop console.log, au contraire c'est plutôt tout ce qui est détails d'événements passés inhabituels (ou habituels) stocké quelque part (fichier/db)